watch-library/simulator/main.c

```diff
@@ -0,0 +1,50 @@
+#include <stdlib.h>
```

Comment:
Big 👍 on MIT license for this contribution. Not sure if it makes sense to put my name though (since it's your project)?

I know a larger company would have contributors sign a CLA, whereby I would be assigning my rights to that company. Not sure how other OSS projects or repositories deal with this or if you have strong feelings about it.

Reply:
Either way is fine! I have to confess I'm mostly annoyed with rules lawyering around licenses, my only concern is that I want the project to stay permissively licensed in case other hardware folks want to fork this as a base for their own watch projects. In short, I don't want a viral license to impede people's ability to do what they want with it. 

I've talked to some colleagues about how they implemented their CLAs, but always bumped into two things: one, I'm less interested in assigning the rights to a company and more interested in making sure it all stays free, and two: I'm worried that a CLA involving rights assignment could impede folks's ability to contribute if they're encumbered in other ways.